packages/vscode-extension/src/handlers.ts

```diff
@@ -273,6 +273,8 @@ async function getIsFromSample() {
 async function getSettingsVersion(): Promise<string | undefined> {
   if (core) {
     const input = getSystemInputs();
+    input.ignoreEnvInfo = true;
```

Comment:
would it be better to move this logic into `core.getProjectConfig` avoid similar issue occurs again?